etc/tool_versioning.yaml

```diff
@@ -31,6 +31,21 @@ toolsets:
                 version: 0.2
                 version_cmd: '{executable} --version'
 
+        1:
+            bcl2fastq:
+                version: v2.20.0.422
+                version_cmd: grep_toolname
+
+            fastq_filterer: null
+
+            fastqc: null
+
+            pigz: null
+
+            seqtk: null
+
+            well_duplicates: null
```

Comment:
Can you add a comment describing how to add tools that enherit from the previous toolset.
I had to go back to the code to understand how it's working

Reply:
There's a test config in `tests/assets/tool_versioning.yaml` with (admittedly brief) descriptions of how the inheritance works